components/dashboard/src/admin/WorkspacesSearch.tsx

```diff
@@ -43,6 +44,11 @@ export function WorkspaceSearch(props: Props) {
     const [queryTerm, setQueryTerm] = useState("");
     const [searching, setSearching] = useState(false);
     const [currentWorkspace, setCurrentWorkspaceState] = useState<WorkspaceAndInstance | undefined>(undefined);
+    const pageLength = 2;
```

Comment:
Why so low? Originally this was set to 100

Reply:
For testing. But I have updated it 50 again.

Reply:
Out of curiosity, why is it needed to commit test code? I'm asking because I'm trying to understand the problem and because this is something worth digging into to eliminate. Committing it risks we actually land it (and we have done so in past and caused incidents).

Reply:
In this case, it was just an overlook by myself.
It could have been a conscious decision  to allow others testing the changes more easily. In that case I would have added a 'hold' and a comment.